ansible/tasks/writeWhiskProperties.yml

```diff
@@ -0,0 +1,8 @@
+---
+# This task will write whisk.properties to the openwhisk_home. 
+# Currently whisk.properties is still needed for consul and tests.
+
+- name: copy whisk.properties template to openwhisk_home
```

Comment:
Want to rename that task as it doesn't copy the template but writes a property file, right?